string name
bool cvWindow
uint8 n_images
---
string person